src/shared/components/cells/CellContext.tsx

```diff
@@ -77,6 +80,19 @@ const CellContext: FC<Props> = ({
     event('editCell button Click')
   }
 
+  const refreshCell = (): void => {
+    const viewWithQueries = view as View<
+      Exclude<ViewProperties, MarkdownViewProperties>
+    >
+    if (
+      Array.isArray(viewWithQueries.properties?.queries) &&
+      viewWithQueries.properties.queries.length
+    ) {
+      resetQueryCacheByQuery(viewWithQueries.properties.queries[0].text)
```

Comment:
why we grabbing the first one here? Is that like the most recent one or something?

Reply:
I'm with Genen on this. Queries here are a bit wonky but TLDR a cell can have multiple queries (multiple tabs) so we would want to refresh both those